ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__ota-nightly-4.14.yaml

```diff
@@ -0,0 +1,91 @@
+base_images:
+  aws-efs-csi-operator-create-efs:
+    name: "4.14"
+    namespace: ocp
+    tag: aws-efs-csi-operator-create-efs
+  cli:
+    name: "4.14"
+    namespace: ocp
+    tag: cli
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  tests-private:
+    name: tests-private
+    namespace: ci
+    tag: "4.14"
+  tests-private-ui:
+    name: tests-private-ui
+    namespace: ci
+    tag: "4.14"
+  tools:
+    name: "4.14"
+    namespace: ocp
+    tag: tools
+  upi-installer:
+    name: "4.14"
+    namespace: ocp
+    tag: upi-installer
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.18
+releases:
+  latest:
+    candidate:
+      architecture: amd64
+      product: ocp
+      stream: nightly
+      version: "4.14"
+resources:
+  '*':
+    limits:
+      memory: 8Gi
+    requests:
+      cpu: "1"
+      memory: 4Gi
+tests:
+- as: aws-ipi-disc-priv-f28-longduration
```

Comment:
I guess we can add these jobs in to `openshift-openshift-tests-private-release-4.N__{amd64|arm64}-nightly.yaml` as what other teams did.

Reply:
yes, we can. considering the job will be used for OTA cases only, so I locate it in verification-tests repo to be consistent with OTA dedicated upgrade jobs.

Reply:
I suggest to add the job to the regular ci, so avoid to maintain another sets of ci yaml for OTA team.

Reply:
Actually since it's for ota team only, i see not too much maintain work here excepting copy/past which the same with upgrade jobs(correct me if my understanding wrong). But putting it in regular ci, i should add 'ota' postfix like other teams to distinguish it from other jobs(at the same time to note other contributor not changing it by chance), then it will be regarded as a `new` e2e jobs which should be handled especially. Is that acceptable?

Reply:
Generally if the job is appended something like `longduration-ota`, generally nobody would change it without any specific reasons. While if we introduced new yaml files, I guess we also need to update RP-results-upload scripts to adopt the new ci yaml files beside coping these files version by version.

Reply:
> we also need to update RP-results-upload scripts to adopt the new ci yaml files

Do u mean the result from new yaml files will not be uploaded if no extra work for it? I don't rmb we did something for it when adding OTA upgrade jobs. Maybe I miss something here.

Reply:
Ignore the comments about RP upload script concerns, I just checked the code, if we did not changed org/repo name, there should be no any update.

Reply:
ah, thanks for confirm. so do u think we could leave the new yamls in current repo?